docs/spec/Status.schema.json

```diff
@@ -17,6 +17,16 @@
                 }
             ]
         },
+        "start_time": {
```

Comment:
Unrelated fix for failing integration test caused by orderly.server response adding start_time to status response - needed to update our schema to match.